ethportal-api/src/dashboard/grafana.rs

```diff
@@ -60,6 +59,13 @@ impl GrafanaAPI {
         let dashboard_json: serde_json::Value = serde_json::from_str(&filled_in_template[..])?;
 
         let dashboard_api_url = format!("{}/{}", self.address, "api/dashboards/db/");
+        // If this is failing with a 409 error, it means that there is likely a problem with the
+        // template, make sure that ...
+        // - All "uid" fields inside a prometheus datasource object contain `"uid": "{prometheus_id}"`
+        // - The root level `"id"` field is set to `null` (`"id": null`)
+        // - The root level `"title"` field is set to "Trin Metrics" (`"title": "Trin Metrics"`)
+        // - The root level `"uid"` field is set to "trin-metrics" (`"uid": "trin-metrics"`)
+
```

Comment:
Nit, the spacing makes it look like the comment is talking about the format! line. Maybe an empty line above, and no empty line below?